.github/workflows/runnable_cxx.yml

```
@@ -0,0 +1,262 @@
+        else
+          echo ::set-env name=CC::${TMP_CC}
+        fi
+
+    # On OSX and Linux, clang is installed by default and in the path,
+    # so make sure ${CC} works
+    - name: '[Posix] Verifying installed clang version'
+      if: matrix.compiler == 'clang' && runner.os != 'Windows'
+      run: |
+        set -e
+        if ${CC} --version | grep -q 'version ${{ matrix.cxx-version }}'; then
+          ${CC} --version
+        else
+            echo "Expected version ${{ matrix.cxx-version }}, from '${CC}', got:"
+            ${CC} --version
+            exit 1
+        fi
+
+    # G++ is only supported on Linux
+    - name: '[Linux] Setting up g++ ${{ matrix.cxx-version }}'
+      if: matrix.compiler == 'g++'
+      run: |
+        # Workaround bug in Github actions
+        curl https://cli-assets.heroku.com/apt/release.key | sudo apt-key add -
+        # Make sure we have the essentials
+        sudo apt-get update
+        sudo apt-get install build-essential software-properties-common -y
+        # This ppa provides multiple versions of g++
+        sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+        sudo apt-get update
+        sudo apt-get install ${{ matrix.target }} ${{ matrix.target }}-multilib
+        echo ::set-env name=CC::${{ matrix.target }}
+
+    # Make sure ${CC} works and we don't use the $PATH one
+    - name: '[Linux] Verifying installed g++ version'
+      if: matrix.compiler == 'g++'
+      run: |
+        set -e
+        if ${CC} --version | grep -q '${{ matrix.target }} (Ubuntu '; then
+          ${CC} --version
+        else
+            echo "Expected version ${{ matrix.target }}, from '${CC}', got:"
+            ${CC} --version
+            exit 1
+        fi
+
+    ########################################
+    #    Building DMD, druntime, Phobos    #
+    ########################################
+    - name: '[Posix] Build compiler & standard library'
+      # By default, VMs have 2 processors available, hence the `-j2`:
+      # https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources
+      run: |
+        # All hosts are 64 bits but let's be explicit
+        ./dmd/src/build.d MODEL=64
+        make -C druntime -f posix.mak -j2 MODEL=64
+        make -C phobos   -f posix.mak -j2 MODEL=64
+        # Both version can live side by side (they end up in a different directory)
+        # However, since clang does not provide a multilib package, only test 32 bits with g++
+        if [ ${{ matrix.compiler }} == "g++" ]; then
+          ./dmd/src/build.d MODEL=32
+          make -C druntime -f posix.mak -j2 MODEL=32
+          make -C phobos   -f posix.mak -j2 MODEL=32
+        fi
+
+    ########################################
+    #        Running the test suite        #
+    ########################################
+    - name: Run C++ test suite
+      run: |
+        ./dmd/test/run.d runnable_cxx MODEL=64
```

The environment should be available in the logs:
```suggestion
        ./dmd/test/run.d --environment runnable_cxx MODEL=64
```

---

This CI might even be a candidate for `TRY_DISABLED` (depending on the average runtime)